var/spack/repos/builtin/packages/savanna/package.py

```diff
@@ -36,13 +36,14 @@ class Savanna(MakefilePackage):
 
     version('develop', git='https://github.com/CODARcode/savanna.git',
             branch='master', submodules=True)
-    version('0.5', '3f13adf29ec30f4acb2ba3fa07ed12b2')
+    version('0.5', git='https://github.com/CODARcode/savanna.git',
+            tag='0.5', submodules=True)
 
     variant('tau', default=False, description='Enable TAU profiling support')
 
     depends_on('mpi')
     depends_on('stc')
-    depends_on('adios +staging')
+    depends_on('adios +fortran +flexpath +dataspaces')
```

Comment:
These are hard requirements, since 'savanna' is the infrastructure for online data staging, and thus must consist of adios built with all its staging methods and fortran support.